File: sdkwork-app-sdk-typescript/src/types/browse-statistics-vo.ts
/** 浏览统计响应 */
export interface BrowseStatisticsVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 总浏览次数 */
  totalCount?: number;
  /** 总浏览时长(分钟) */
  totalDuration?: number;
  /** 平均浏览时长(秒) */
  averageDuration?: number;
  /** 今日浏览次数 */
  todayCount?: number;
  /** 本周浏览次数 */
  weeklyCount?: number;
  /** 本月浏览次数 */
  monthlyCount?: number;
  /** 浏览类型分布 */
  typeDistribution?: Record<string, number>;
  /** 浏览来源分布 */
  sourceDistribution?: Record<string, number>;
  /** 每日浏览趋势(最近7天) */
  dailyTrend?: Record<string, number>;
}

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/BrowseStatisticsVO.kt
package com.sdkwork.app

data class BrowseStatisticsVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val totalCount: Int? = null,
    val totalDuration: Int? = null,
    val averageDuration: Int? = null,
    val todayCount: Int? = null,
    val weeklyCount: Int? = null,
    val monthlyCount: Int? = null,
    val typeDistribution: Map<String, Int>? = null,
    val sourceDistribution: Map<String, Int>? = null,
    val dailyTrend: Map<String, Int>? = null
)
